ui/components/config/provider-form.tsx

```
@@ -760,6 +772,73 @@ export default function ProviderForm({ provider, onSave, onCancel, existingProvi
                               </div>
                             )}
 
+                            {/* Bedrock Key Configuration */}
+                            {selectedProvider === 'bedrock' && (
+                              <div className="space-y-4 pt-2">
+                                <div>
+                                  <label className="text-sm font-medium">Access Key (Required)</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)